Sources/Kitura/RouterRequest.swift

```diff
@@ -103,12 +98,21 @@ public class RouterRequest {
     public internal(set) var matchedPath = ""
 
     /// The original URL as a string.
-    public var originalURL: String {
-        return serverRequest.urlString
-    }
+    /// This contains just the path and query parameters starting with '/'
+    /// Use "urlComponents" for the full URL
+    @available(*, deprecated, message:
+        "This contains just the path and query parameters starting with '/'. use 'urlComponents' instead")
+    public var originalURL : String { return serverRequest.urlString }
```

Comment:
This isn't quite true. urlComponents will change through a request as parts of the path are dropped off in sub-routers. originalURL is the complete original URL before anything happened.